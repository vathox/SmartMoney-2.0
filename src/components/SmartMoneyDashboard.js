import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummary from './ExpensesSummary'

const SmartMoneyDashboard = () =>
    <div>
        <ExpensesSummary/>
        <ExpenseListFilters/>
        <ExpenseList/>
    </div>
;

export default SmartMoneyDashboard