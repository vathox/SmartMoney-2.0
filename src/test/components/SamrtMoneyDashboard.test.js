import React from 'react';
import SmartMoneyDashboard from '../../components/SmartMoneyDashboard'
import { shallow } from 'enzyme';

test('should render Dashboard correctly', () => {
    const wrapper = shallow(<SmartMoneyDashboard/>);
    expect(wrapper).toMatchSnapshot();
});