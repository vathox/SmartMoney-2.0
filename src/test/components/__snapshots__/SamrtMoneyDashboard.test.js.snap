// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
</div>
`;
